src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,77 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
```

Comment:
Do you mean in the interface or config file? Config-file-wise you can set any value you want and it will work but not in the interface because it uses the selector.